### DISCUSSION TITLE
有没有对路由功能支持比较好的 GUI 推荐啊？

### DISCUSSION THREAD

**Discussion by FlyGinger** [v2fly/v2ray-core#1683] [Q&A]
Qv2ray 停止更新之后，强迫症就开始焦虑了，一直在寻找替代品，但是似乎其他 GUI 都是偏向于 PAC 而不是路由。

有没有什么支持路由的 GUI 推荐啊，最好还支持 Trojan 协议？

**Comment by ChenSino** [comment]
使用v2raya吧

**Reply by FlyGinger** [reply]
找到最后没找到我自己写了一个……

[K2Ray](https://github.com/FlyGinger/K2Ray)

**Reply by ChenSino** [reply]
> 找到最后没找到我自己写了一个……
> 
> [K2Ray](https://github.com/FlyGinger/K2Ray)

还没支持Linux啊，用的什么工具开发的桌面啊？ 我以为是electron点进去好像不是

**Reply by FlyGinger** [reply]
没支持 linux ，按道理可以支持，毕竟是跨平台的框架，但是我没有 linux 设备。

用的是 tauri + vue ，比 electron 体积小了好多。原来用 electron 写的，总共没几个功能 86MB ，现在用 tauri 重构只有 3MB 了。

**Reply by ChenSino** [reply]
> 没支持 linux ，按道理可以支持，毕竟是跨平台的框架，但是我没有 linux 设备。
> 
> 用的是 tauri + vue ，比 electron 体积小了好多。原来用 electron 写的，总共没几个功能 86MB ，现在用 tauri 重构只有 3MB 了。

ok,我去看看tauri自己打包试试。

**Reply by FlyGinger** [reply]
还要改代码才行，因为比如设置系统代理什么的是系统相关的命令，所以代码里有一些类似 `switch (os.platform)` 之类的代码，现在还没有 linux 。

**Reply by ChenSino** [reply]
> 还要改代码才行，因为比如设置系统代理什么的是系统相关的命令，所以代码里有一些类似 `switch (os.platform)` 之类的代码，现在还没有 linux 。

哦哦。好的。
